src/components/static/Navigation.svelte

```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
+      <div class="hidden text-white lg:block xl:block 2xl:block h-96" style="margin-left: 12.5%; margin-right: 12.5%;">
```

Comment:
xl and 2xl sections not needed. h-96 might also be a bit much, h-80 seems like it should be big enough.